spec/integration/file_content_type_validation_integration_spec.rb

```diff
@@ -364,6 +364,11 @@ class Person
       before { subject.avatar = '' }
       it { is_expected.to be_valid }
     end
+
+    context 'invalid string' do
+      before { subject.avatar = '\'' }
```

Comment:
We need a better example of invalid JSON. Yes `\'` is an invalid JSON. Can we come up with a better invalid JSON which looks more like a JSON object but invalid?

Reply:
I can imagine a more complex example but what's the need to retest what is valid or invalid JSON when I can guess every variations is already part of the json gem test suite?

Reply:
For the little tale, I choose this example because that very simple but invalid JSON string is exactly what a nasty bot tried to send to one of my Rails app just a few days ago and what raised the JSON::ParserError that leads me to write this pull request.

Reply:
Understood. A simple invalid JSON will do. The current example is too simple IMHO.

Reply:
I changed it